install_config/configuring_openstack.adoc

```diff
@@ -161,8 +168,7 @@ integration will not work.
 ====
 When triggering a containerized installation, only the directories of
```

Comment:
So is this note, `containerized installation` is a deprecated concept. No matter rpm-installed node or system-container installed node, the cloud config file should always be `/etc/origin/cloudprovider/{{ openshift_node_group_cloud_provider }}.conf`